hazelcast/include/hazelcast/client/internal/nearcache/NearCacheManager.h

```diff
@@ -120,6 +120,13 @@ namespace hazelcast {
                      * Destroys all defined {@link NearCache} instances.
                      */
                     void destroyAllNearCaches();
+
+                    /**
+                     * Lists all existing {@link NearCache} instances.
+                     *
+                     * @return all existing {@link NearCache} instances
+                     */
+                    std::vector<boost::shared_ptr<BaseNearCache> > listAllNearCaches();
```

Comment:
We have similar method as commented out above ?

Reply:
Exactly, was unimplemented. I just deleted those lines.